### PR TITLE
Feat/sys 3623 opengov introduction

### PR DESCRIPTION
**Pull request by ivan-cholakov**
## Proposed changes

Implemented OpenGov in main runtime

## Type of change/Merge

🚨What type of change is this PR? </br>
_Put an `x` in the boxes that apply_

- [x] Release 
  - [x] Increase versions 
  - [x] Baseline tests passed  
  - Release type:
    - [ ] Major release 
    - [x] Minor release 
    - [ ] Patch release 

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x] You describe the purpose of the PR, e.g.:
  - What does it do?
  - Highlight what important points reviewers should know about;
  - Indicates if there is something left for follow-up PRs.
- [ ] Documentation updated
- [x] Business logic tested successfully
- [ ] Verify First, Write Last: In Substrate development, it is important that you always ensure preconditions are met and return errors at the beginning. After these checks have completed, then you may begin the function's computation.

### CODE REVIEW

**Review by alexpArtos — APPROVED** [review]

File: runtime/avn/src/governance/mod.rs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+impl pallet_referenda::Config for Runtime {
+    type WeightInfo = pallet_referenda::weights::SubstrateWeight<Runtime>;
+    type RuntimeCall = RuntimeCall;
+    type RuntimeEvent = RuntimeEvent;
+    type Scheduler = Scheduler;
+    type Currency = Balances;
+    type SubmitOrigin = frame_system::EnsureSigned<AccountId>;
+    type CancelOrigin = EitherOf<EnsureRoot<AccountId>, ReferendumCanceller>;
+    type KillOrigin = EitherOf<EnsureRoot<AccountId>, ReferendumKiller>;
+    type Slash = ();
```
Comment (thadouk) [inline runtime/avn/src/governance/mod.rs:53]: Does this mean that slash are disabled?

File: runtime/avn/src/governance/tracks.rs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            min_approval: APP_REFERENDUM_CANCELLER,
+            min_support: SUP_REFERENDUM_CANCELLER,
+        },
+    ),
+    (
+        4,
+        pallet_referenda::TrackInfo {
+            name: "referendum_killer",
+            max_deciding: 1000,
+            decision_deposit: 2000 * AVT,
```
Comment (thadouk) [inline runtime/avn/src/governance/tracks.rs:104]: Have we confirmed this amounts for these operations with the team & business? I assume yes.

File: runtime/avn/src/governance/tracks.rs [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        } else if let Ok(custom_origin) = origins::Origin::try_from(id.clone()) {
+            match custom_origin {
+                origins::Origin::WhitelistedCaller => Ok(1),
+                // General admin
+                origins::Origin::GeneralAdmin => Ok(2),
+                // Referendum admins
+                origins::Origin::ReferendumCanceller => Ok(3),
+                origins::Origin::ReferendumKiller => Ok(4),
+            }
+        } else {
```
Comment (thadouk) [inline runtime/avn/src/governance/tracks.rs:137]: why do we need two matches here? One would suffice. It feels odd.
Is it to group the system and custom origins conceptually?

**Review by thadouk — APPROVED** [review]

File: runtime/avn/src/governance/tracks.rs (hunk 3 above)
Comment (ivan-cholakov) [reply runtime/avn/src/governance/tracks.rs:137]: yes it is

**Pull request merged**